resources/views/backend/layouts/partials/side_bar.blade.php

```diff
@@ -1,26 +1,26 @@
 <div class="col-md-3 left_col">
-  <div class="left_col scroll-view">
-    <div class="navbar nav_title" style="border: 0;">
-      <a href="index.html" class="site_title"><i class="fa fa-paw"></i> <span>{!! trans('labels.promotion_admin') !!}</span></a>
-    </div>
+    <div class="left_col scroll-view">
+        <div class="navbar nav_title" style="border: 0;">
+            <a href="index.html" class="site_title"><i class="fa fa-paw"></i> <span>{!! trans('labels.promotion_admin') !!}</span></a>
+        </div>
+
+        <div class="clearfix"></div>
 
-    <div class="clearfix"></div>
 
-    
-    <!-- /menu profile quick info -->
+        <!-- /menu profile quick info -->
 
-    <br />
+        <br />
 
-    <!-- sidebar menu -->
-    <div id="sidebar-menu" class="main_menu_side hidden-print main_menu">
-      <div class="menu_section">
-        <ul class="nav side-menu">
-          <li><a href="{{ url('admin/dashboard') }}"><i class="fa fa-home"></i> {!! trans('labels.dasboard') !!} </a>               
-          </li>
-        </ul>
-      </div>
+        <!-- sidebar menu -->
+        <div id="sidebar-menu" class="main_menu_side hidden-print main_menu">
+            <div class="menu_section">
+                <ul class="nav side-menu">
+                    <li><a href="{!! url('admin/dashboard') !!}"><i class="fa fa-home"></i> {!! trans('labels.dasboard') !!} </a></li>
+                    <li><a href="{!! url('admin/city') !!}"> <i class="fa fa-taxi"></i> {!! trans('labels.city') !!} </a></li>
```

Comment:
`url()` is OK but it's better if you use the `route` function, it uses the route name. 
```php
route('admin.city.index');
```